pkg/backingstore/reconciler.go

```diff
@@ -177,16 +176,6 @@ func (r *Reconciler) Reconcile() (reconcile.Result, error) {
 		}
 	}
 
-	oldStatefulSet := &appsv1.StatefulSet{}
```

Comment:
Are we sure there is no option that old systems are still out there? If only the print is the issue maybe we should change to print in case we do find the old stateful set format instead? like to run kubecheckquiet...

Reply:
I do not see an option that we have that old of STS structure.

Reply:
ok, as long as we have this documented